target/linux/ath79/dts/ar7242_buffalo_wzr-300hp-b0a1.dts

```diff
@@ -0,0 +1,243 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "ar7242.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+	compatible = "buffalo,wzr-300hp-b0a1", "qca,ar7242";
+	model = "Buffalo WZR-300HP B0A1";
+
+	aliases {
+		led-boot = &led_diag;
+		led-failsafe = &led_diag;
+		led-upgrade = &led_diag;
+	};
+
+	extosc: ref {
+		compatible = "fixed-clock";
```

Comment:
Please remove this node and `&pll`. it has been moved to SoC dtsi. https://github.com/openwrt/openwrt/commit/520c90854ca73eb6c3d8feeda59766c90bdd4144